### PR TITLE
Beryllium lnwallet 202220331

### PR DESCRIPTION
**Pull request by eoliveros**
- new channel opener page with bootbox.
- fixed ln-channel balance in ln_wallet/  page
- verify that the value required in the invoice.html page
- flash success message is not green in pay_invoice endpoint.
- fixed the amount of satoshis used when opening new channel.
- fixed the invoices.html to show the value as sats instead of msat.

### CODE REVIEW

File: ln_wallet_endpoint.py [hunk 1]
```diff
@@ -160,26 +160,22 @@ def decode_bolt11(bolt11=None):
...
-        flash(Markup(e.args[0]), 'danger')
-    return render_template("lightning/channel_opener.html")
+    if request.method == 'POST':
+        amount = request.form["amount"]
+        nodeid = request.form["nodeid"]
+        try:
+            rpc = LnRpc()
+            rpc.connect_node(nodeid)
+            node_id = nodeid.split("@")
+            # pylint: disable=unused-variable
```
Comment (djpnewton) [inline ln_wallet_endpoint.py:173]: get rid of this comment and just remove the `result = ` in the next line

File: templates/lightning/invoice.html [hunk 2]
```diff
@@ -60,6 +60,9 @@ <h2>Create Invoice</h2>
     }
 
     function confirm(form_class, amount) {
+	if (!document.querySelector(form_class).reportValidity()) {
```
Comment (djpnewton) [inline templates/lightning/invoice.html:63]: fix spacing, (you are probably mixing spaces and tabs)

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: ln_wallet_endpoint.py (hunk 1 above)
Comment (eoliveros) [reply ln_wallet_endpoint.py]: removed the result = and the comment.

File: templates/lightning/invoice.html (hunk 2 above)
Comment (eoliveros) [reply templates/lightning/invoice.html]: changed tab to space(s)

**Pull request merged**